Project-1/Peihong_Xie_Project_1_50375781.md

```diff
@@ -0,0 +1,249 @@
+  ```(a)  B(x,y) that says that x is a brother of y
+  (b)  A(x,y) that says that x is an aunt of y
+  (c)  C(x,y) that says that x and y are cousins 
+  (d)  O(x) that says that x is an only child
+  (e)  T(x) that says that x has exactly two brothers 
+```
+
+Answer:
+
+(a) B(x,y)=def. ∃u(P(u,x)∧P(u,y))∧¬F(x)∧¬(x=y)
+
+(b) A(x,y)=def. ∃u∃w(P(u,y)∧P(w,x)∧P(w,u)∧¬(x=u))∧F(x)
+
+(c) C(x,y)=def. ∃u∃w∃z(P(w,z)∧P(w,u)∧P(z,x)∧P(u,y)∧¬(z=u))∧¬(x=y)
+
+(d) O(x)=def. ∃u(P(u,x)∧∀y(P(u,y)→x=y))
+
+(e) T(x)=def. ∃u∃y∃z(¬(y=z)∧¬(x=y)∧¬(x=z)∧P(u,x)∧P(u,y)∧P(u,z)∧¬F(y)∧¬F(z))∧∀w(P(u,w)∧¬F(w)→ w=x ∨ w=y ∨ w=z)
+
+[4] Let V be a vocabulary of the attribute (concept) language with complements (ALC) consisting of a role name "parent_of" and a concept name "Male". Interpret parent_of as "x is a parent of y" and M as "x is male". Where possible define the following formulas in this vocabulary; where not possible, explain why: 
+  ```(a)  B that says that x is a brother of y
+  (b)  A that says that x is an aunt of y
+  (c)  C that says that x and y are cousins 
+  (d)  O that says that x is an only child
+  (e)  T that says that x has exactly two brothers 
+```
+
+Answer:
+
+(a) $B ≡ M\sqcap ∃parentOf^-.(∃parentOf\ge2)$
+
+(b) $A ≡ ¬M\sqcap ∃parentOf^-.((∃parentOf.(∃parentOf.M\sqcup ¬M))\sqcap∃parentOf\ge2)$
+
+(c) $C ≡ ∃parentOf^-.(∃parentOf^-.(\ge2parentOf.(parentOf.M\sqcup ¬M)))$
+
+(d) $O ≡ ∃parentOf^-.(∃parentOf\le1)$
+
+(e) $T ≡ (M\sqcap ∃parentOf^-.(\le3∃parentOf.M \sqcap \ge3∃parentOf.M))\sqcup(¬M\sqcap∃parentOf^-.(\le2∃parentOf.M \sqcap \ge2∃parentOf.M))$
+
+[5] Select two formulas defined in ALC from question 4 to form the basis of a T-Box. Supplement this T-box with whatever other axioms you like, as well as an A-box, so that you ultimately construct a knowledge base K = (T,A). Provide a _model_ of K. This may be graphical or symbolic or both. 
+
+Answer:
+
+$K_1 = (T_1,A_1)$ such that:
+
+T-box $T_1$={$B ≡ M\sqcap∃parentOf^-.(∃parentOf\ge2)$, 
+
+$O≡∃parentOf^-.(∃parentOf\le1)$, 
+
+$GP≡∃parentOf.(∃parentOf.M\sqcup ¬M)$}
+
+A-box $A_1$={(Mary,Karl):ParentOf, Karl:Male}.
+
+The following is a model $\Im=(\bigtriangleup^\Im,.^\Im)$ of $K_1$:
+
+$\bigtriangleup^\Im$={a,b,c}
+
+$M^\Im$={a}
+
+$B^\Im=\emptyset$
+
+$O^\Im$={b,a}
+
+$GP^\Im$={c}
+
+$parentOf^\Im$={(c,b),(b,a)}
+
+$Karl^\Im$={a}
+
+$Mary^\Im$={b}
+
+$Jack^\Im$={c}
+
+[6] Explain the difference - using natural language - between the first-order prefixes:
+  ```(a) ∃x∀y and ∀x∃y
+  (b) ∃x∀y∃z and ∀x∃y∀z 
+  (c) ∀x∃y∀z∃w and ∃x∀y∃z∀w
+```
+Answer:
+
+(a) ∃x∀y means “there exists x such that for all y” while ∀x∃y means "for all x, there exists y such that"
+
+(b) ∃x∀y∃z means “there exists x such that for all y, there exists z such that” while ∀x∃y∀z means "for all x, there exists y such that for all z"
+
+(c) ∀x∃y∀z∃w means "for all x, there exists y such that for all z, there exists w such that" while ∃x∀y∃z∀w means "there exists x such that for all y, there exists z such that for all y"
```

Comment:
I think you meant "for all w" at the end of line 172

Reply:
> I think you meant "for all w" at the end of line 172

Thanks!